svc/pkg/cluster/src/ops/datacenter/get.rs

```diff
@@ -78,7 +78,7 @@ impl TryFrom<DatacenterRow> for Datacenter {
 pub async fn cluster_datacenter_get(ctx: &OperationCtx, input: &Input) -> GlobalResult<Output> {
 	let datacenters = ctx
 		.cache()
-		.fetch_all_json("cluster.datacenters", input.datacenter_ids.clone(), {
+		.fetch_all_json("cluster.datacenters2", input.datacenter_ids.clone(), {
 			let ctx = ctx.clone();
```

Comment:
Presumably we add the 2 to signify that the datacenter has been migrated?

Reply:
Adding the 2 makes it use a different cache key because `cluster.datacenters` stores proto which would cause a deserialization error when reading as json